src/main/eo/org/eolang/txt/text.eo

```diff
@@ -36,8 +36,15 @@
 
   s > @
 
+  [start len] > slice
+    text > @
+      ^.s.slice start len
+
   # Trim it from both sides
-  [] > trim /string
+  [] > trim
+    text > @
+      tr
+    [] > tr /string
```

Comment:
@levBagryansky I think we don't need to change the name of the atom, just return `EOtext()` in java code will be enough. WDYT?

Reply:
@Graur agree, I'll try change to this variant